shared/desktop/app/node.desktop.tsx

```diff
@@ -213,6 +221,19 @@ const findRemoteComponent = (windowComponent: string, windowParam: string) => {
 const plumbEvents = () => {
   Electron.app.on('KBkeybase' as any, (_: string, action: Action) => {
     switch (action.type) {
+      case 'activeChanged':
+        // the installer reads this file to understand the gui state to not interrupt
+        // TODO change how this works
+        try {
+          fs.writeFileSync(
```

Comment:
Would this remove the other data in this file? Mine has the window location/size info there.

Reply:
yeah, but thats historical and ignored. thats since moved to gui_config.json and owned by the daemon